src/pages/NewChairConsole.js

```diff
@@ -151,6 +190,32 @@ const NewChairConsole = () => {
     }
   };
 
+  const createElection = (darId, index) => {
+    if (darId !== null) {
+      let copy;
+      const i = index + ((currentPage - 1) * tableSize);
+      Election.createDARElection(darId)
+        .then((electionResponse) => electionResponse.json())
+        .then((newElection) => {
+          Notifications.showSuccess({text: "Election successfully opened"});
+          copy = cloneDeep(filteredList);
+          copy[parseInt(i)].election = newElection;
+          setFilteredList(copy);
```

Comment:
Alright now you'll need to update the electionList right after the filterList update. Remember that the filteredList is derived from applying filters on the electionList, so filteredList is more of a temporary state (changes with filters) while the electionList can be considered as a source of truth. However, because filteredList is a subset of electionList, you can't use the index you've calculated to access electionList. Instead you'll need to search electionList for the proper record.

One thing to note is that you can't search based on elections since you might be creating one for the first time. However the dar the election is associated is a viable search target.

Reply:
right, thank you!!